llvm/lib/IR/Core.cpp

```diff
@@ -2921,6 +2921,17 @@ int LLVMGetSignednessInfo(LLVMValueRef Inst) {
   }
 }
 
+/*-- Get the variable in the llvm.dbg.declare instructions .................--*/
+LLVMValueRef LLVMGetDbgDeclareVar(LLVMValueRef Inst) {
```

Comment:
Also, can this function be grouped with `LLVMGetSignednessInfo` in the same category: 

``` c
/*--.. Get data from debugging information ........................................--*/
```

or something similar?